### ISSUE TITLE
[Docs][Sampson]Missing datafile for viszulation example

### ISSUE THREAD

**Issue by sara-02** [networkx/networkx#3692]
How does one obtain the `sampson_data.zip` and `samplike*.txt` files mentioned in the visualization example here: https://networkx.github.io/documentation/stable/auto_examples/drawing/plot_sampson.html
Running the `.py` or the `.ipynb` raises 
```
FileNotFoundError: [Errno 2] No such file or directory: 'sampson_data.zip'
```
Networkx: 2.4 installed via pip running python3

**Comment by sara-02** [comment]
I found the file under https://github.com/networkx/networkx/blob/master/examples/drawing/sampson_data.zip.
Like the Knuth Miles example[1] Can we add line in the sampson example as well, to point to the zip file location?

1: https://networkx.github.io/documentation/stable/auto_examples/drawing/plot_knuth_miles.html

**Comment by jarrodmillman** [comment]
I will look into whether this should be added as part of sphinx-gallery or whether we should just add it ourselves.  Either way, I will fix this before nx 2.5.
